applications/DamApplication/custom_processes/dam_chemo_mechanical_aging_young_process.hpp

```diff
@@ -0,0 +1,219 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Lorenzo Gracia
+//
+//
+
+#if !defined(KRATOS_DAM_CHEMO_MECHANICAL_AGING_YOUNG_PROCESS )
+#define  KRATOS_DAM_CHEMO_MECHANICAL_AGING_YOUNG_PROCESS
+
+#include <cmath>
+
+// Project includes
+#include "includes/kratos_flags.h"
+#include "includes/kratos_parameters.h"
+#include "processes/process.h"
+
+// Application include
+#include "dam_application_variables.h"
+
+namespace Kratos
+{
+
+class DamChemoMechanicalAgingYoungProcess : public Process
+{
+    
+public:
+
+    KRATOS_CLASS_POINTER_DEFINITION(DamChemoMechanicalAgingYoungProcess);
+    
+//----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+    /// Constructor
+    DamChemoMechanicalAgingYoungProcess(ModelPart& rmodel_part,
+                                Parameters& rParameters
+                                ) : Process(Flags()) , mrModelPart(rmodel_part)
+    {
+        KRATOS_TRY
+			 
+        //only include validation with c++11 since raw_literals do not exist in c++03
+        Parameters default_parameters( R"(
+            {
+                "model_part_name":"PLEASE_CHOOSE_MODEL_PART_NAME",
+                "mesh_id": 0,
```

Comment:
I'm not sure if it is necessary now to pass the mesh_id. Perhaps @loumalouomega can tell us this...